packages/edit-site/src/components/global-styles/font-library-modal/style.scss

```diff
@@ -66,6 +72,11 @@
 	.font-library-modal__font-card__count {
 		color: $gray-700;
 	}
+
+	.font-library-modal__font-variant_demo-text {
+		white-space: nowrap;
+		text-overflow: ellipsis;
+	}
```

Comment:
> By the way, fonts shouldn't wrap

`white-space: nowrap` has already been [applied](https://github.com/WordPress/gutenberg/blob/d8215e974b743d6486a1d6972818aed6e989409c/packages/edit-site/src/components/global-styles/font-library-modal/style.scss#L78) to this text. I think the root cause is due to `text-wrap: pretty` added in #60164.

https://github.com/WordPress/gutenberg/assets/54422211/6d9eff2e-42d1-4e2a-9bc6-ac3e01455a5c

Since this CSS selector already exists [here](https://github.com/WordPress/gutenberg/blob/d8215e974b743d6486a1d6972818aed6e989409c/packages/edit-site/src/components/global-styles/font-library-modal/style.scss#L77), can't we just add `text-wrap: unset` there?

Reply:
After looking at this further, `text-wrap: unset` doesn't unset this, only `white-space: nowrap` or `text-wrap: nowrap`. I've also noticed that `white-space: nowrap` was already set on these elements, but it I think the change in https://github.com/WordPress/gutenberg/pull/60164 made this CSS not specific enough.

Instead of my original change, I've made the CSS for `.font-library-modal__font-variant_demo-text` and `.font-library-modal__font-variant_demo-image` more specific, which seems to have worked well. I moved `.font-library-modal__font-variant_demo-image` as well as it didn't feel correct to move one without the other. This is handled in https://github.com/WordPress/gutenberg/pull/60570/commits/6bb02a943fb182599ffb5fb8795f688446c14fd7.